internal/cli/apis.go

```diff
@@ -324,6 +326,52 @@ auth0 apis delete <id>`,
 	return cmd
 }
 
+func openApiCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Open API settings page in Auth0 Manage",
+		Long:    "Open API settings page in Auth0 Manage.",
+		Example: "auth0 apis open <id>",
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := apiID.Pick(cmd, &inputs.ID, cli.apiPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			// Heuristics to determine if this a valid ID, or an audience value
+			if _, err := url.ParseRequestURI(inputs.ID); err == nil || len(inputs.ID) != 24 {
```

Comment:
what's `24`coming from ? is inputs.ID a url? in that case I wonder if we can make it more explicit on the command help

Reply:
24 is the string length of the ID. Audiences are usually URLs, but not necessarily, so there is that additional heuristic to help disambiguate. I'll make it more explicit on the command help and add a comment explaining the values, thanks!

Reply:
Done in https://github.com/auth0/auth0-cli/pull/253/commits/62c64d535f533809886487d159ef00c33b04881b